projects/control-service/projects/settings.gradle

```diff
@@ -4,7 +4,7 @@ pluginManagement {
       id 'io.spring.dependency-management' version '1.0.11.RELEASE'
       id 'com.palantir.docker' version '0.34.0'
       // https://projectlombok.org/setup/gradle
-      id 'io.freefair.lombok' version '6.5.1'
+      id 'io.freefair.lombok' version '6.1.0-m3'
```

Comment:
Should we troubleshoot the reason? Should we temp workaround with looking for the last operable version?

Reply:
Yes, we should troubleshoot the reason. But first thing is to bring main CICD to stable.